.gitignore

```diff
@@ -47,4 +47,5 @@ dist/
 *.http
 
 # Reverse-engineering data
-data/
\ No newline at end of file
+data/
+.DS_Store
```

Comment:
could you add another header here explaining what `.DS_Store` is?

Reply:
It's a Mac filesystem thing that has stuff like where the folder is dragged on the desktop and its size, etc